core/controllers/admin.py

```diff
@@ -71,6 +76,114 @@ class AdminHandler(base.BaseHandler):
     """Handler for the admin page."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'action': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': [
+                        'reload_exploration', 'reload_collection',
+                        'generate_dummy_explorations', 'clear_search_index',
+                        'generate_dummy_new_structures_data',
+                        'generate_dummy_new_skill_data',
+                        'save_config_properties', 'revert_config_property',
+                        'upload_topic_similarities',
+                        'regenerate_topic_related_opportunities',
+                        'update_feature_flag_rules'
+                    ]
+                },
+                'default_value': None
```

Comment:
@DubeySandeep the reason behind using default here is that, in the backend it is expected 'clear_search_index' should come with action field of payload, but from frontend there is no payload sent while clearing the search index.
Please guide me.

Reply:
Yes, there seems to be some be some mistake. I think we should add the action 'clear_search_index' to the payload of [clearSearchIndexAsync](https://github.com/oppia/oppia/blob/462e812a2c0e4cc04f5089f2b32462a661225e77/core/templates/domain/admin/admin-backend-api.service.ts#L317).

Reply:
@rohitkatlaa do I have to fix this ? or should I create an issue for other contributors?

Reply:
I think it is better to create an issue and handle this in another PR after confirming the details.

Reply:
If it's a minor fix then I would suggest doing it in the PR. [The major reason for having a schema validation project is to make the handler's structure better and avoid such mistakes. In general, if you see such an issue, it would be great if you can fix those.]

Reply:
Thanks @DubeySandeep. So @Nik-09 please do the necessary changes in this PR. Thanks.

Reply:
@Nik-09 Please do create an issue because the error doesn't seem to be trivial and further analysis might be required.